crates/liboci-cli/src/features.rs

```diff
@@ -0,0 +1,7 @@
+use clap::Parser;
```

Comment:
May I ask you to update the README for liboci-cli?
https://github.com/containers/youki/tree/main/crates/liboci-cli